lib/livebook_web/components/form_components.ex

```diff
@@ -20,6 +20,26 @@ defmodule LivebookWeb.FormComponents do
 
   attr :rest, :global, include: ~w(autocomplete readonly disabled step min max)
 
+  def text_field(%{type: "base_url"} = assigns) do
```

Comment:
Let's make it a separate component, can be very specific like `schemaless_url_field` :)

In Livebook we use separate components (as opposed to what Phoenix generates), because we have more fields and they have different attributes (and different content anyway), so I think that's more maintainable. In this case the attributes are the same, but I still believe it makes sense for consistency :)